_security/access-control/api.md

```diff
@@ -1297,6 +1297,85 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
+### Check configuration upgrade
+Introduced 2.13
+{: .label .label-purple }
+
+Checks the configuration bundled with the Security plugin against the existing security configuration, returns if an upgrade can be performed and what resources would be updated.
+
+#### Request
+
+```json
+GET _plugins/_security/api/_upgrade_check
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "status" : "OK",
+  "upgradeAvailable" : true,
+  "upgradeActions" : {
+    "roles" : {
+      "add" : [ "flow_framework_full_access" ]
+    }
+  }
+}
+```
+
+#### Response fields
+
+| Field    | Data type  | Description                   |
+|:---------|:-----------|:------------------------------|
+| `upgradeAvailable`   | Boolean     | True when an upgrade can be performed.  |
+| `upgradeActions`  | Object     | The security objects that would be modified by an upgrade.  |
+
+### Perform configuration upgrade
+Introduced 2.13
+{: .label .label-purple }
+
+Adds and updates resources on the existing security configuration from the configuration bundled with the Security plugin.
+
+These bundled configuration files can be found in the `opensearch-project/security/config` directory.  Default configuration files are updated on OpenSearch upgrade, whereas the cluster configuration is only updated by the cluster operators. This request used to bridge these scenarios.
```

Comment:
Shouldn't the path be `<OPENSEARCH_HOME>/config/opensearch-security/`? It may be different in previous major versions.

Reply:
nit: Change `This request used to bridge these scenarios.` to `This request is used to source new security configuration defaults like additions or changes to reserved roles.`

Reply:
Reworded this a little - think I got the heart of your concern addressed feel free to reopen/resuggest